src/html.ts

```diff
@@ -1004,7 +1004,7 @@ export type OptionTagAttrs = DefaultTagAttrs & {
     label?: string
     selected?: boolean
     text?: string
-    value?: string
+    value?: string | null
```

Comment:
This section of the file is auto generated based on the built-in DOM APIs. We can't change it. Probably the best way to handle it would be to replace null with undefined in `optionsForSelect`